dietpi/dietpi-software

```diff
@@ -5647,15 +5647,14 @@ _EOF_
 
 		if To_Install 195 # youtube-dl
 		then
-			aDEPS=('python3')
-			Download_Install 'https://yt-dl.org/downloads/latest/youtube-dl' /usr/local/bin/youtube-dl
-			G_EXEC chmod +x /usr/local/bin/youtube-dl
+			# Remove obsolete youtube-dl and youtube-dl-py2 we used previously
+			G_EXEC rm -f /usr/local/bin/youtube-dl{,-py2}
 
-			# youtube-dl supports Python 2 and Python 3, but its shebang calls Python 2, else fails: https://github.com/ytdl-org/youtube-dl/issues/27649
-			G_EXEC sed -i '1s/python$/python3/' /usr/local/bin/youtube-dl
+			aDEPS=('python3' 'ffmpeg')
```

Comment:
Why is FFmpeg required? If so, then it should be installed as dependency on software ID 7 (which is FFmpeg).

Reply:
As stated in [dependencies](https://github.com/yt-dlp/yt-dlp#dependencies). 

> While all the other dependencies are optional, ffmpeg and ffprobe are highly recommended
> 
> Required for [merging separate video and audio files](https://github.com/yt-dlp/yt-dlp#format-selection) as well as for various [post-processing](https://github.com/yt-dlp/yt-dlp#post-processing-options) tasks.

I'll update dependencies then.